adapter/src/test/java/org/jvalue/ods/adapterservice/datasource/TestHelper.java

```diff
@@ -40,7 +50,15 @@ public static Datasource generateParameterizableDatasource(
       true,
       new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX").parse("1905-12-01T02:30:00.123Z"),
       50000L);
-      DatasourceSchema schema = new DatasourceSchema("{a:1}");
+
+    String jsonObject = "{\"test\":1}";
+    ObjectMapper objectMapper = new ObjectMapper();
+    Object schema = null;
+    try { 
+      schema  = objectMapper.readValue(jsonObject, new TypeReference<Map<String,Object>>(){});
+    } catch (Exception e) {
+      //TODO: handle exception
```

Comment:
Probably just throw a runtime exception?
`throw new RuntimeException("Could not parse schema string to object")`